docs/refs/public-api.md

```diff
@@ -0,0 +1,12 @@
+# Public API
+The public API of OLM v1 is as follows:
+
+- Kubernetes APIs. For more information on these APIs, see:
+    - [operator-controller API reference](./api/operator-controller-api-reference.md)
+    - [catalogd API reference](./api/catalogd-api-reference.md)
+- `catalogd` webserver. For more information on what this includes, see the [catalogd webserver documentation](./api/catalogd-webserver.md)
```

Comment:
Since this section is pretty short, I think you could create an Additional resources section at the end of the page for the links. Maybe after line 12?

```
## Additional resources
- Operator Controller API reference
- Catalogd API reference
- Catalogd web server

Reply:
Would you mind elaborating on what value this would bring to a reader over the current setup?

Reply:
To me, the links are a little distracting where they are. The point you are trying to get across is to define the scope of the public API. I think that if you break out the links to the API references at the end of the page, it is a little tidier. But this is definitely a nit, and feel free to ignore it doesn't feel relevant to you.

Reply:
As a heuristic for determining whether something should be an inline link vs an additional resource, I ask myself if the reader needs to stop and read the link before continuing.

In the case of a prerequisite or context information that is vital to understanding a concept or completing a task, then I use an inline link. If the reader can follow up on the information later or if the link is provided to for helpful context, then I break out the link in an additional resources section.

Reply:
As long as the _content_ is OK - i'd like to get this merged in so we at least have it and can keep progressing towards our v1.0.0 release. We can always tweak this in the future!